### DISCUSSION TITLE
How to assign label to indicator values?

### DISCUSSION THREAD

**Discussion by bigbenbeer** [kernc/backtesting.py#444] [Q&A]
I have a number of indicators that consist of more than one value. Let us take MACD as an example. My MACD function returns the MACD and the signal lines and the following code snippet I place within my `init(self)` function.
```
# MACD Indicator
        self.macdSignal, self.macd = self.I(
            macd,
            self.data.Close,
        )
```
However, when this indicator is placed on the plot, both the macdSignal and macd plots have the same value in the legend.

![image](https://user-images.githubusercontent.com/24241587/131600622-c09fe9e5-bf07-44a0-9085-61a0f2d4ed56.png)

How can I change the name of each line?

**Comment by kernc** [comment]
See issue https://github.com/kernc/backtesting.py/issues/382.
